src/test/WETH.t.sol

```diff
@@ -104,6 +105,17 @@ contract WETHTest is DSTestPlus {
         assertEq(weth.totalSupply(), depositAmount - withdrawAmount);
     }
 
+    function testReentrancy() public {
```

Comment:
can we name this something like testReentrancyFails

Reply:
done